### ISSUE TITLE
State claimer on help channels

### ISSUE THREAD

**Issue by aphkyle** [python-discord/meta#168]
### What feature are you proposing?

Edit the "Available help channel" embed 
![image](https://user-images.githubusercontent.com/81857274/150662919-9f9389e4-34fb-4568-b635-8ba0b4079c12.png)
to
![image](https://user-images.githubusercontent.com/81857274/150663135-f653a7f5-3750-4594-bce6-255cd483401d.png)

### What is your justification for the proposed feature?

I've been visiting some help channels,
I found that some claimers delete their messages. Then, someone else who isn't the claimant came and asked questions. Adding this feature to make helpers know who the claimer is.

**Comment by Qwerty-133** [comment]
I don't think we should change anything other than the title and the first sentence there, the other information should still be available for the claimant to read through.

**Comment by aphkyle** [comment]
> I don't think we should change anything other than the title and the first sentence there, the other information should still be available for the claimant to read through.

Perhaps keeping the "Remember to ..." part

**Comment by minalike** [comment]
Yes let's keep all that content, as it's still relevant to the claimant and people helping might want to copy/paste some of those guidelines to further prompt the claimant to ask their question, and also quickly grab a link to the "asking good questions" guide.

I noticed yesterday that mentions in embeds do not show up in Discord searches under `mentions:<ID>`. From a moderation perspective I think being able to do this search would be a benefit, but it's not a dealbreaker.
